prestodev/spark3.0-iceberg/Dockerfile

```
@@ -0,0 +1,23 @@
+FROM prestodev/centos7-oj11:unlabelled
+
+ARG SPARK_VERSION=3.0.0
+ARG HADOOP_VERSION=2.7
+ARG ICEBERG_VERSION=0.9.0
+
+ARG SPARK_ARTIFACT="spark-${SPARK_VERSION}-bin-hadoop${HADOOP_VERSION}"
+
+ENV SPARK_HOME=/spark
+
+RUN wget "https://archive.apache.org/dist/spark/spark-${SPARK_VERSION}/${SPARK_ARTIFACT}.tgz"; \
```

`set -xeu;`

(otherwise wget or tar error will be ignored)

---

Thanks, I missed that.